.genx/templates/angular/grid.hbs

```diff
@@ -1,6 +1,6 @@
     <rapid-grid-pro
-      header-case-type="capitalCase"
-      only-template-col-defs
+      header-case-type="capitalCase"{{#if config.gridOptions?.columnDefs}}
+      only-template-col-defs{{/if}}
```

Comment:
I tested every case and none of them returns: `only-template-col-defs`

The reason for this is that  `gridOptions.columnDefs`/`columns` is serialized earlier in the file: `.genx/utils/formatRouteData.js` so we don't have access to this object via handlebars (we can only render them as is now `{{{ config.gridOptions }}})`

GridOptions can have columnsDefs and onRowClicked - from the point of view of genesis-create there is no way to have only onRowClicked, there will always be columnDefs with or without onRowClicked, so we could only check if gridOptions exists.

But then we will have a bug if someone uses it via terminal and will add only onRowClicked without columnDefs (edge case) 

so theoretically we could not serialize it in formatRouteData but later in the template to have access to it all the time by creating e.g. custom handlebar helper
```
{{#if tile.config.gridOptions}}
export const gridOptions: GridOptionsConfig = {{{ serializeGridOptions config.gridOptions }}}
{{/if}}
```

but it requires a bit more investigation - there might be a better way to do it

Reply:
after a moment's thought we could also do something like this since we have a new component structure:

```
{{#if config.gridOptions}}
?only-template-col-defs=${() => !!gridOptions?.columnDefs}
{{/if}}
```

It should be simpler

Reply:
I fixed it for web components, but I need to look at the angular implementation... ⏲️

Reply:
Update: added ```useOnlyTemplateCols``` to Handlebars and used it in a condition